### PR TITLE
[onert-micro] Support Fill kernel

### PR DESCRIPTION
**Pull request by BalyshevArtem**
This pr adds Fill kernel with unit test

for issue: #10709

ONE-DCO-1.0-Signed-off-by: Artem Balyshev <a.balyshev@samsung.com>

### CODE REVIEW

File: onert-micro/luci-interpreter/include/luci_interpreter/test_models/fill/FillKernel.h [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+{
+/*
+ * Fill Kernel:
+ *
+ *     Dims(3, 2)      Input(scalar)
+ *           \            /
+ *                Fill
+ *                 |
+ *                 |
+ *              Output(3, 2)
```
Comment (BalyshevArtem) [inline onert-micro/luci-interpreter/include/luci_interpreter/test_models/fill/FillKernel.h:36]: Test with Dims const input = {3, 2}, and Value is input scalar

**Review by SlavikMIPT — APPROVED** [review]
LGTM

**Comment by chunseoklee** [comment]
@BalyshevArtem If there is any code from other opensource (e.g. here from TF), please add license statement as in  https://github.com/Samsung/ONE/pull/10718. If not, just ignore it.

**Comment by BalyshevArtem** [comment]
@chunseoklee 
@SlavikMIPT 
Add license line

**Pull request merged**